release_notes/368-release-notes.adoc

```diff
@@ -81,23 +82,31 @@ In previous versions of {product-title}, the `write` permission for the `APIToke
 Beginning with {product-title-short} 3.68, Red Hat has updated the image version number convention for the `Scanner`, `Scanner DB`, and `Collector` images. The version numbers for these images now match the version numbers for the `Main` image.
 ====
 
+[IMPORTANT]
+====
+Due to the change in image repository names in `registry.redhat.io`, customers who mirror ACS
+   image repositories should verify their mirroring is set up to mirror images from these
+   new locations.
+====
+
 |===
 | Image | Description | Current version
 
 | Main
 | Includes Central, Sensor, Admission Controller, and Compliance.
 Also includes `roxctl` for use in continuous integration (CI) systems.
-| registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:{rhacs-version}
+| `registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:3.68.0`
 
 | Scanner
 | Scans images and nodes.
-| registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:{scanner-version}
+| `registry.redhat.io/advanced-cluster-security/rhacs-scanner-rhel8:3.68.0`
 
 | Scanner DB
 | Stores image scan results and vulnerability definitions.
-| registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:{scanner-version}
+| `registry.redhat.io/advanced-cluster-security/rhacs-scanner-db-rhel8:3.68.0`
 
 | Collector
 | Collects runtime activity in Kubernetes or {ocp} clusters.
-| registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:{collector-version}
+| `registry.redhat.io/advanced-cluster-security/rhacs-collector-rhel8:3.68.0`
+  `registry.redhat.io/advanced-cluster-security/rhacs-collector-slim-rhel8:3.68.0`
```

Comment:
@gaurav-nelson I wanted to capture both "collector" and "collector-slim" somehow in this table because technically these are different image repositories now. The "best" thing I could come up with is to put them both here. Maybe a better way is possible but I don't know.